src/bgcl.js

```diff
@@ -457,6 +464,33 @@ BGCL.prototype.handleFee = co(function *handleFee(opts) {
   opts.field('feePerKb', feeInfo.feePerKb.toString().bold);
 });
 
+/**
+ * Create a new key pair for the given key
+ * @param {object} opts The arguments being passed to the function
+ * @param {string} [opts.args.coin] The coin to use instead of the session coin
+ */
+BGCL.prototype.handleNewKey = co(function *handleFee(opts) {
+  const coin = opts.args.coin || opts.session.coin;
+  const params = opts.correctParams(opts.args);
+
+  if (opts.args.coin) {
+    if (!isValidCoin(opts.args.coin)) {
+      throw new Error(`${coin} is an invalid cointype for selected environment -> ${opts.bitgo.getEnv()}`);
+    }
+    delete params.coin;
+  }
+
+  const seed = crypto.randomBytes(32);
```

Comment:
Don't think you need to pass in external entropy - if you just don't pass any seed, it will create a random one.

Reply:
Should I pass entropy or the seed? I was thinking about making the seed an optional argument so ppl can recover their pub and priv based on that